components/telemetry-operator/internal/validation/variable_validator.go

```diff
@@ -56,7 +56,7 @@ func validateMandatoryFieldsAreEmpty(vr telemetryv1alpha1.VariableReference) boo
 func findConflictingVariables(logPipeLine *telemetryv1alpha1.LogPipeline, vr telemetryv1alpha1.VariableReference, existingPipelineName string) error {
 	for _, v := range logPipeLine.Spec.Variables {
 		if v.Name == vr.Name {
-			return fmt.Errorf("variable with name '%s' has a been previously used in pipeline '%s'", v.Name, existingPipelineName)
+			return fmt.Errorf("variable with name '%s' has been previously used in pipeline '%s'", v.Name, existingPipelineName)
```

Comment:
Maybe we can shorten it a bit?

```suggestion
			return fmt.Errorf("variable '%s' has been previously used in pipeline '%s'", v.Name, existingPipelineName)
```

Is the problem that it's been used before (no matter whether it's still in use now), or is it the problem that it's used in two places at the same time?

```suggestion
			return fmt.Errorf("Cannot use variable name '%s', already used in pipeline '%s'", v.Name, existingPipelineName)
```
Not sure whether I may change the syntax like this; will it be weird in rendered version?

Reply:
not sure if second message is a bit hard tone. We want to have a bit softer tone if I understand correct. I will go with first one.